code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -351,6 +351,14 @@
 	var/show_descs = show_descriptors_to(user)
 	if(show_descs)
 		msg += "<span class='notice'>[jointext(show_descs, "<br>")]</span>"
+
+	var/list/human_examines = decls_repository.get_decls_of_subtype(/decl/human_examination)
+	for(var/exam in human_examines)
+		var/decl/human_examination/HE = human_examines[exam]
+		var/adding_text = HE.do_examine(user, distance, src)
+		if(adding_text)
+			msg += adding_text
```

Comment:
If you wanna optimize, you could compact this down to:
```
msg += HE.do_examine(user, distance, src) || null
```
But it doesn't really matter. (And the `|| null` is onlt there to keep it from adding "0" if a 0 is returned)